Gems/ImGui/Code/Include/LYImGuiUtils/HistogramContainer.h

```diff
@@ -10,8 +10,8 @@
 
 #ifdef IMGUI_ENABLED
 #include <AzCore/std/containers/deque.h>
+#include <AzCore/std/string/string.h>
```

Comment:
It seems odd that we need to pull this header in when no code has changed in the header, but I see that AZStd::string is used here.